src/main/java/eu/enexa/service/ContainerManager.java

```diff
@@ -17,26 +17,34 @@ public interface ContainerManager {
 
     /**
      * Stop the container with the given ID.
-     * 
+     *
      * @param containerId
      * @return TODO what should be returned, here?
      */
     String stopContainer(String containerId);
 
     /**
      * Get the status of the container with the given ID as String.
-     * 
+     *
      * @param containerId the ID of the container for which the status should be
      *                    returned.
      * @return The status as String or {@code null} if an error occurs or the
      *         container does not exist
      */
     String getContainerStatus(String containerId);
 
+    /**
+     * this test method will rewritten , this set the host paths
+     * @param sharedDirectory
+     * @param experimentWriteablePathDirectory
+     * @param modulePathDirectory
+     */
+    void setHostPaths(String sharedDirectory, String experimentWriteablePathDirectory, String modulePathDirectory);
```

Comment:
I would claim that this method shouldn't be necessary at all as part of this interface. To me, it looks like a clear sign that something is wrong here :thinking: